src/collector/ServicesClient.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { retry } from '@lifeomic/attempt';
+import nodeFetch, { Request } from 'node-fetch';
+
+import { retryableRequestError, fatalRequestError } from './error';
+import { URLSearchParams } from 'url';
+
+export interface ServicesClientInput {
+  apiToken: string;
+}
+
+interface QueryParam {
+  [param: string]: string | string[];
+}
+
+const BASE_URL = 'https://api.bugcrowd.com/';
+
+/**
+ * Services Api
+ * https://docs.bugcrowd.com/reference
+ */
+export class ServicesClient {
+  readonly baseUrl: string;
+  readonly authHeader: { [key: string]: string };
+
+  constructor(config: ServicesClientInput) {
+    this.baseUrl = BASE_URL;
+    this.authHeader = {
+      Authorization: `Token ${config.apiToken}`,
+      Accept: 'application/vnd.bugcrowd+json',
+    };
+  }
+
+  test(): Promise<any[]> {
```

Comment:
There are warnings in the GitHub Actions about this being an `any`. I'd recommend `Promise<unknown>`, since you don't care about the return value. In fact, it may be best to just avoid returning anything since the method is not meant to return anything that is to be used, but only to verify there is no failure to authenticate.